libs/csm/rest/csm_rest_quota.py

```diff
@@ -116,32 +125,32 @@ def verify_get_set_user_quota(self, uid, payload, verify_response=False,
                     if key in ("enabled","max_size","max_objects","check_on_raw"):
                         continue
                     if key == "max_size_kb":
-                        payload.update({"max_size_kb":max_size/1000})
+                        payload.update({"max_size_kb":get_response["max_size"]/1000})
                     self.log.info("Actual response for %s: %s", key, payload[key])
                     if value != payload[key]:
                         self.log.error("Actual and expected response for %s didnt match", key)
                         result = False
         else:
             self.log.error("Status code check failed.")
             result = False
-        return result, resp
+        return result, get_response
```

Comment:
Can u set the result variable for IF in the line 114 and also u need to return response based on what is failed.

Reply:
I couldn't completely understand what is being expected in this comment.

Reply:
Return doesnt take care of the scenario when set has failed

Reply:
okay modified